src/pages/ModelDetails/ModelDetails.js

```diff
@@ -437,10 +437,17 @@ const ModelDetails = () => {
                       emptyStateMsg={"There are no relations in this model"}
                     />
                     {shouldShow("relations-title", activeView) && (
-                      <h5>
-                        Cross-model relations (
-                        {consumedTableRows.length + offersTableRows.length})
-                      </h5>
+                      <>
+                        {consumedTableRows.length > 0 ||
```

Comment:
There is quite a bit of logic here that's easy to get wrong in some updates, can you add a test for this plz

Reply:
The problem is the test data dump does not support CMR?